src/nunit.gui.core/Views/IMainView.cs

```diff
@@ -36,8 +47,8 @@ public interface IMainView : IView
         FormWindowState WindowState { get; set; }
         Font Font { get; set; }
 
-        event FormClosingEventHandler FormClosing;
-        event DragEventHandler DragDrop;
+        event MainViewClosingEvent MainViewClosing;
```

Comment:
This can be `event CommandHandler Closing` 

`CommandHandler` is our view event handler with no args.`
`Closing` is sufficient as the name since you can only refer to it using an instance of the main view.